modules/microshift-config-cli-manifests.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-config-cli-manifests_{context}"]
+= Using CLI tools and creating manifests
```

Comment:
I can't find this file in the preview

Reply:
Commented out for now, likely not used in production docs.